pytorch/training/docker/1.6.0/py3/Dockerfile.gpu

```diff
@@ -0,0 +1,186 @@
+# The tag for the base image is: 10.1-cudnn7-devel-ubuntu16.04
+FROM nvidia/cuda@sha256:229f04d85d723f45461d1155c8f106ac27afd48b9c9c58d8f94fc38e367d5596
```

Comment:
Is there a way we can unpin this sha and set to 10.1-cudnn7-devel-ubuntu16.04 again?

Reply:
+1

Reply:
Unpinned sha in next update.